sdk/tools/ci.yml

```
@@ -0,0 +1,27 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - /sdk/tools/azure-sdk-archetype/
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - /sdk/tools/azure-sdk-archetype/
+
+extends:
+  template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
```

```suggestion
  template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
```

Do we think that the client template will work for this? There might be some library checks that could fail.

---

The pipelines did run successfully, so, `client` template seems to work.